<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget {
	background-color:  qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0.0297872 rgba(255, 255, 255, 0), stop:0.0893617 rgba(255, 255, 255, 25), stop:0.12766 rgba(255, 255, 255, 59), stop:0.170213 rgba(255, 255, 255, 101), stop:0.225532 rgba(255, 255, 255, 137), stop:0.285106 rgba(255, 255, 255, 177), stop:0.348936 rgba(255, 255, 255, 231), stop:0.421277 rgba(255, 255, 255, 255));
}
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="ClickableLabel" name="profileLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="client.qrc">:/resources/main_profile_icon.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ClickableLabel" name="chatLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>70</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="client.qrc">:/resources/main_chat_icon.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ClickableLabel" name="settingLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="client.qrc">:/resources/main_setting_icon.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="pageEmpty">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="client.qrc">:/resources/welcome_label.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageProfile">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>200</number>
       </property>
       <property name="topMargin">
        <number>70</number>
       </property>
       <property name="rightMargin">
        <number>200</number>
       </property>
       <property name="bottomMargin">
        <number>70</number>
       </property>
       <item row="0" column="0">
        <widget class="QStackedWidget" name="profileStackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page_3">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="ProfileWidget" name="profileWidget" native="true">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_4">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="EditProfileWidget" name="profileEditWidget" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageChat">
      <layout class="QGridLayout" name="gridLayout_7">
       <property name="leftMargin">
        <number>50</number>
       </property>
       <property name="topMargin">
        <number>30</number>
       </property>
       <property name="rightMargin">
        <number>50</number>
       </property>
       <property name="bottomMargin">
        <number>30</number>
       </property>
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QTabWidget::pane { 
	border: 0px solid rgb(245, 245, 245);
	border-radius: 30px;
}

QTabWidget::tab-bar {
	left: 10px; 
}
QTabWidget{
	background-color: rgb(245, 245, 245);
}

QTabBar::tab {
	margin-left:4px;
	padding:4px;
	border-top-left-radius: 6px;
	border-top-right-radius: 6px;
	background: rgb(245, 245, 245);
	border: 3px solid rgb(245, 245, 245);
	color: rgb(60, 47, 50);
	min-height:30px;
}

QTabBar::tab:selected {
	border-color: rgb(60, 47, 50);
	border-bottom-color: rgb(60, 47, 50); 
	margin-bottom:-1px;
}</string>
         </property>
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="tabChats">
          <property name="styleSheet">
           <string notr="true">QWidget#tabChats {
	background-color: rgb(245, 245, 245);
	border-radius: 10px;
}</string>
          </property>
          <attribute name="icon">
           <iconset resource="client.qrc">
            <normaloff>:/resources/chat_chat_icon.png</normaloff>:/resources/chat_chat_icon.png</iconset>
          </attribute>
          <attribute name="title">
           <string>Чаты</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="UsersList" name="usersList" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>320</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>320</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="ChatWidget" name="chatWidget" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabGroups">
          <property name="styleSheet">
           <string notr="true">QWidget {
	background-color: rgb(245, 245, 245);
	border-radius: 10px;
}</string>
          </property>
          <attribute name="icon">
           <iconset resource="client.qrc">
            <normaloff>:/resources/chat_group_icon.png</normaloff>:/resources/chat_group_icon.png</iconset>
          </attribute>
          <attribute name="title">
           <string>Группы</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="0">
            <widget class="ChatList" name="chatList" native="true">
             <property name="maximumSize">
              <size>
               <width>320</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="GroupWidget" name="groupWidget" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabChannels">
          <property name="styleSheet">
           <string notr="true">QWidget {
	background-color: rgb(245, 245, 245);
	border-radius: 10px;
}</string>
          </property>
          <attribute name="icon">
           <iconset resource="client.qrc">
            <normaloff>:/resources/chat_channel_icon.png</normaloff>:/resources/chat_channel_icon.png</iconset>
          </attribute>
          <attribute name="title">
           <string>Каналы</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="ChannelList" name="channelList" native="true">
             <property name="minimumSize">
              <size>
               <width>320</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>320</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="ChannelWidget" name="channelWidget" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageSetting">
      <layout class="QGridLayout" name="gridLayout_6">
       <property name="leftMargin">
        <number>250</number>
       </property>
       <property name="topMargin">
        <number>70</number>
       </property>
       <property name="rightMargin">
        <number>250</number>
       </property>
       <property name="bottomMargin">
        <number>70</number>
       </property>
       <item row="0" column="0">
        <widget class="SettingWidget" name="settingWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>clickablelabel.h</header>
  </customwidget>
  <customwidget>
   <class>ProfileWidget</class>
   <extends>QWidget</extends>
   <header>profilewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SettingWidget</class>
   <extends>QWidget</extends>
   <header>settingwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EditProfileWidget</class>
   <extends>QWidget</extends>
   <header>editprofilewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UsersList</class>
   <extends>QWidget</extends>
   <header>userslist.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChatWidget</class>
   <extends>QWidget</extends>
   <header>chatwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChannelList</class>
   <extends>QWidget</extends>
   <header>channellist.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChannelWidget</class>
   <extends>QWidget</extends>
   <header>channelwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChatList</class>
   <extends>QWidget</extends>
   <header>chatlist.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GroupWidget</class>
   <extends>QWidget</extends>
   <header>groupwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="client.qrc"/>
 </resources>
 <connections/>
</ui>
